lib/ibm_watson/visual_recognition_v4.rb

```diff
@@ -280,7 +288,8 @@ def update_collection(collection_id:, name: nil, description: nil)
 
       data = {
         "name" => name,
-        "description" => description
+        "description" => description,
+        "training_status" => training_status
```

Comment:
These should be readonly and not generated. need to look at generator

Reply:
Just to clarify, Are you saying I need to add hand edits to remove training_status?
It is also passed in so remove it as a parameter to?

    # @param training_status [TrainingStatus] Training status information for the collection.
    # @return [IBMCloudSdkCore::DetailedResponse] A `IBMCloudSdkCore::DetailedResponse` object representing the response.
    def update_collection(collection_id:, name: nil, description: nil, training_status: nil)

Reply:
Yes, training status should be read only and not a param in the method signature. We will discuss this in standup.